services/apps/portal/src/application/application-parcel/application-parcel.dto.ts

```diff
@@ -38,6 +41,9 @@ export class ApplicationParcelDto {
   @AutoMap(() => String)
   ownershipTypeCode?: string | null;
 
+  @AutoMap(() => String)
```

Comment:
I think this should not be String since its an object, AutoMap(() => [ApplicationParcelOwnershipTypeDto])

Reply:
Good catch! I'll remove it. It is not going to work anyways since I have forMember in automapper